modules/op-about-tasks.adoc

```diff
@@ -46,3 +46,10 @@ spec: <4>
 This Task starts the pod and runs a container inside that pod using the `maven:3.6.0-jdk-8-slim` image to run the specified commands. It receives an input directory called `workspace-git` that contains the source code of the application.
 
 The Task only declares the placeholder for the Git repository, it does not specify which Git repository to use. This allows Tasks to be reusable for multiple Pipelines and purposes.
+
+[WARNING]
+====
+{pipelines-title} 1.3 and earlier versions in Technology Preview (TP) create a task without verifying the xref:../authentication/managing-security-context-constraints.adoc[Security Context Constraints] (SCC) of users. This allows any authenticated user to create a task with a container running with a privileged SCC.
```

Comment:
Suggest:
{pipelines-title} 1.3 and earlier versions in Technology Preview (TP) allowed users to create a task without verifying their...(SCC).

To avoid the usage of 'with a' in the same sentence, how about: Thus, any authenticated user could create a task using a container running with a privileged SCC.